hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtTargetResourceTest.java

```diff
@@ -853,7 +853,56 @@ public void getActionWithEmptyResult() throws Exception {
     }
 
     @Test
-    @Description("Ensures that the expected response is return when update was cancelled.")
+    @Description("Ensures that the expected response is returned for update action.")
```

Comment:
check if there is a test present for getUpdtaeAction and check if maintenance window os not present